app/models/post.rb

```diff
@@ -1,3 +1,13 @@
 class Post < ApplicationRecord
   belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_post_counter
+    user.increment!(:PostsCounter)
```

Comment:
- [ ] Hello and nice job on the blog app, but your diagram is having an `AuthorId` in your `post` instead of the user.

Reply:
Thanks for your review. I have fixed that bug.